pkg/crc/preflight/preflight_checks_darwin.go

```diff
@@ -6,27 +6,39 @@ import (
 	"golang.org/x/sys/unix"
 	"io"
 	"net/http"
+	neturl "net/url"
 	"os"
 	"os/exec"
 	"os/user"
+	"path"
 	"path/filepath"
+	"strings"
+	"syscall"
 
+	"github.com/code-ready/crc/pkg/crc/constants"
 	"github.com/code-ready/crc/pkg/crc/logging"
 	"github.com/code-ready/crc/pkg/crc/oc"
+	dl "github.com/code-ready/crc/pkg/download"
 	crcos "github.com/code-ready/crc/pkg/os"
 )
 
 const (
 	virtualBoxDownloadURL   = "https://download.virtualbox.org/virtualbox/6.0.4/VirtualBox-6.0.4-128413-OSX.dmg"
 	virtualBoxMountLocation = "/Volumes/VirtualBox"
 
+	hyperkitDownloadURL       = "https://657-55985023-gh.circle-artifacts.com/0/hyperkit"
+	hyperkitDriverDownloadURL = "https://teuf.fedorapeople.org/crc-driver-hyperkit"
```

Comment:
@cfergeau this will be replaced with proper release we do on the driver side right? Anything stoping to do that release?

Reply:
Oh thanks for pointing this out, I had totally forgotten about it. Yep, definitely should be replaced by something more official. I have no idea what the workflow is to make a release, so that would be the main blocker ;) Apart from this, I think the current crc-driver-hyperkit git master is good for release.

Reply:
I've now made a crc-driver-hyperkit release, attached a crc-driver-hyperkit binary to it, and I've also added the hyperkit binary there  for now.